packages/plugin/src/modules/fetchData.js

```diff
@@ -17,7 +16,6 @@ export const fetchData = async ({
 
     const extraOptions = {
         dashboard: forDashboard,
-        noData: { text: i18n.t('No data') },
```

Comment:
Is it safe to remove this...?

Reply:
Yes, I'm setting that as the default in the analytics lib: here: https://github.com/dhis2/analytics/pull/438/files#diff-ed9ecb480b8d52ffa9ceedc0e9988581R39-R44